docs/hub/spaces-changelog.md

```diff
@@ -1,5 +1,10 @@
 # Spaces Changelog
 
+
+## [2022-12-07] - Add support for Streamlit 1.15
```

Comment:
Also add another for the custom sleep time

Reply:
bc2a6f111d5f06a097f8a82cf3654cd8612974ea